config/initializers/google_drive_vars.rb

```diff
@@ -6,5 +6,6 @@
   vacancy_publish_feedback: ENV['AUDIT_VACANCY_PUBLISH_FEEDBACK_WORKSHEET_GID'],
   general_feedback: ENV['AUDIT_GENERAL_FEEDBACK_WORKSHEET_GID'],
   interest_expression: ENV['AUDIT_EXPRESS_INTEREST_WORKSHEET_GID'],
-  subscription_creation: ENV['AUDIT_SUBSCRIPTION_CREATION_WORKSHEET_GID']
+  subscription_creation: ENV['AUDIT_SUBSCRIPTION_CREATION_GID'],
```

Comment:
Merge / rebase error: This shouldn't have been renamed.

Reply:
Gaaah. Fixed now